content/id/docs/tasks/run-application/force-delete-stateful-set-pod.md

```diff
@@ -0,0 +1,81 @@
+---
+reviewers:
+title: Hapus Paksa Pod StatefulSet
+content_type: task
+weight: 70
+---
+
+<!-- overview -->
+Laman ini menjelaskan bagaimana cara menghapus Pod yang menjadi bagian dari sebuah {{< glossary_tooltip text="stateful set" term_id="StatefulSet" >}}, dan menjelaskan pertimbangan yang harus diperhatikan saat melakukannya.
+
+
+## {{% heading "prerequisites" %}}
+
+
+* Ini merupakan tugas yang cukup rumit dan memiliki potensi melanggar beberapa properti yang melekat dari StatefulSet.
+* Sebelum melanjutkan, pastikan kamu paham dengan pertimbangan yang disebutkan di bawah ini.
+
+
+
+<!-- steps -->
+
+## Pertimbangan StatefulSet
+
+Pada operasi normal dari StatefulSet, **tidak pernah** ada kebutuhan untuk menghapus paksa sebuah Pod StatefulSet. [Pengontrol StatefulSet](/docs/concepts/workloads/controllers/statefulset/) bertanggung jawab terhadap pembuatan, _scaling_ dan penghapusan terhadap anggota dari StatefulSet. Pengontrol akan berusaha menjaga agar jumlah Pod yang ditentukan dari 0 hingga N-1 hidup dan siap sedia. StatefulSet memastikan bahwa, pada waktu kapanpun, akan ada minimal satu Pod dengan identitas yang telah ditetapkan berjalan pada klaster. Hal ini direferensikan sebagai semantik *at most one* yang disediakan StatefulSet.
+
+Penghapusan paksa secara manual harus dilakukan dengan hati-hati, karena hal tersebut berpotensi melanggar semantik _at most one_ yang melekat pada StatefulSet. StatefulSet dapat digunakan untuk menjalankan aplikasi terklaster dan terdistribusi yang membutuhkan identitas jaringan dan penyimpanan yang stabil dan tetap. Aplikasi-aplikasi ini biasanya memiliki konfigurasi yang tergantung dengan sejumlah anggota dengan identitas yang tetap. Memiliki banyak anggota dengan identitas yang sama berpotensi menjadi bencana dan dapat menimbulkan kehilangan data (contohnya skenario  _split brain_ pada sistem berbasis kuorum).
```

Comment:
“Bencana” seems to much.
```suggestion
Penghapusan paksa secara manual harus dilakukan dengan hati-hati, karena hal tersebut berpotensi melanggar semantik _at most one_ yang melekat pada StatefulSet. StatefulSet dapat digunakan untuk menjalankan aplikasi terklaster dan terdistribusi yang membutuhkan identitas jaringan dan penyimpanan yang stabil dan tetap. Aplikasi-aplikasi ini biasanya memiliki konfigurasi yang tergantung dengan sejumlah anggota dengan identitas yang tetap. Memiliki banyak anggota dengan identitas yang sama berpotensi menjadi masalah dan dapat menimbulkan kehilangan data (contohnya skenario  _split brain_ pada sistem berbasis kuorum).
```

Reply:
I'll change to `berpotensi menimbulkan kerusakan dan kehilangan data`